src/game/InstanceData.h

```diff
@@ -104,10 +104,10 @@ class LOOKING4GROUP_IMPORT_EXPORT InstanceData : public ZoneScript
         virtual void ResetEncounterInProgress();
 
         //Called when a player successfully enters the instance.
-        virtual void OnPlayerEnter(Player *) {}
+        void OnPlayerEnter(Player *);
```

Comment:
"virtual" is necessary for zulaman,shattered halls,old hillsbrad, sethek halls, blood furnace, hyjal summit